glotaran/project/result.py

```diff
@@ -170,47 +262,70 @@ def markdown(self, with_model: bool = True, base_heading_level: int = 1) -> Mark
         return MarkdownStr(result_table)
 
     def _repr_markdown_(self) -> str:
-        """Special method used by ``ipython`` to render markdown."""
+        """Return a markdown representation str.
+
+        Special method used by ``ipython`` to render markdown.
+
+        Returns
+        -------
+        str
+            The scheme as markdown string.
+        """
         return str(self.markdown(base_heading_level=3))
 
     def __str__(self):
+        """Overwrite of ``__str__``."""
```

Comment:
On the matter of adding docstring for builtin functions ... to [add docstring or not to add docstrings](https://softwareengineering.stackexchange.com/a/303382/176402) @glotaran/maintainers ? 
In any case `def __str__(self):` -> `def __str__(self) -> str:`